lms/static/js/certificates/views/certificate_whitelist.js

```diff
@@ -32,7 +32,12 @@
                 render: function(){
                     var template = this.loadTemplate('certificate-white-list');
                     this.$el.html(template({certificates: this.collection.models}));
-
+                    if(this.collection.isEmpty()){
+                        this.$("#generate-exception-certificates").addClass("is-disabled");
+                    }
+                    else{
+                        this.$("#generate-exception-certificates").removeClass("is-disabled");
```

Comment:
Really, this should be a separate PR; it's not clear how it's related to the title of this PR. _At_ _the_ _least_, it needs to be a separate commit.